samples/sensor/tmp116/sample.yaml

```diff
@@ -3,8 +3,7 @@ sample:
 tests:
   sample.sensor.tmp116:
     harness: console
-    platform_allow: nucleo_f401re
-    integration_platforms:
```

Comment:
Why is this removed?

Reply:
no need to add integration_platforms if this sample is already targeting one platform only.